tests/testthat/_snaps/find.md

```diff
@@ -27,3 +27,11 @@
     Message
       0 conflicts
 
+# using canonical reference
+
+    Code
+      conflict_scout(c("testthat", "dplyr", "pipe"))
+    Message
+      1 conflict
+      * `%>%()`: magrittr and pipe
```

Comment:
This seems a bit confusing to me since you're reporting the name of a package that the user hasn't attached.

Reply:
Yeah, and conflicted doesn't seem to behave well in this case too:

``` r
library(conflicted)
library(dplyr)
library(testthat)
library(fastpipe) # moodymudskipper/fastpipe

conflict_scout()
#> 4 conflicts
#> • `%>%()`: fastpipe and magrittr
#> • `filter()`: dplyr and stats
#> • `lag()`: dplyr and stats
#> • `matches()`: testthat and tidyselect
conflicts_prefer(magrittr::`%>%`)
#> [conflicted] Will prefer magrittr::`%>%` over any other package.
environment(`%>%`)
#> <environment: namespace:fastpipe>
```

<sup>Created on 2023-09-27 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>